glib/src/value.rs

```diff
@@ -205,6 +205,21 @@ pub unsafe trait FromValue<'a>: Sized {
     unsafe fn from_value(value: &'a Value) -> Self;
 }
 
+/// Similar to `FromValue` but works for owned `Value.
+///
+/// ```compile_fail
+/// let x = "gtk-rs".to_value();
+///
+/// let y = x.get_owned::<&str>();
+/// ```
+pub unsafe trait FromOwnedValue: for<'a> FromValue<'a> {
+    /// Get the contained value from a `Value`
+    ///
+    /// # Safety
+    /// `Self::Checker::check()` must be called first and must not fail.
+    unsafe fn from_value_owned(value: &Value) -> Self;
```

Comment:
```suggestion
    unsafe fn from_value_owned(value: &Value) -> Self {
        <Self as FromValue>::from_value(v)
    }
```

Does that work? Then you don't need to implement any functions anywhere

Reply:
Indeed, thanks!